driver-mct/cime_config/config_component_e3sm.xml

```diff
@@ -372,7 +372,6 @@
       <value compset="_DATM.*_SLND.*MPAS" grid="oi%oRRS15to5">96</value>
       <value compset="_DATM.*_SLND.*MPAS" grid="oi%oARRM60to10">96</value>
       <value compset="_DATM.*_SLND.*MPAS" grid="oi%oARRM60to6">96</value>
-      <value compset="_DATM.*_SLND.*MPAS" grid="oi%ARRM10to60">96</value>
```

Comment:
you are removing this one for now because we want to Hailong first, right?

Reply:
this came in [1199ea1](https://github.com/darincomeau/E3SM/pull/26/commits/1199ea104fb74e37f8b937e589909ce2087a0e77) to change the default timestep back to 30 minutes for G-cases from our discussions with Robert a few weeks ago.

But these leaves in place `ATM_NCPL=96` for the `arcticx4v1` atm mesh on this branch (but we haven't opened the companion PR into E3SM master yet - and needs discussion as noted).